projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts

```diff
@@ -99,7 +96,6 @@ describe("Tabbing order - tests don't require user to be logged in", () => {
 
   describe('Header Sub Categories - Desktop', () => {
     context('Brands', () => {
-      // TODO: Test currently fails because navigation-ui works incorrectly
```

Comment:
Is it fixed already? If not, please leave this comment here

Reply:
Yes, that particular e2e desktop test is now fixed.